test/utils.hpp

```diff
@@ -191,6 +209,19 @@ std::pair<bool, double> compareEqual(DDataType const* deviceD,
     return std::make_pair(retval, max_relative_error);
 }
 
+template <typename DDataType>
+double getEpsilon()
+{
+    if(std::is_same_v<DDataType, _Float16>)
+    {
+        return 0.0009765625; // numeric_limits<_Float16>::epsilon() => 0
```

Comment:
Should we include the std::numeric_limits<_Float16> extension from rocWMMA?

Reply:
yeah, definitely. I will copy type related files from wmma to hiptensor.

Reply:
- Copied xfloat32, type_traits and types_ext from wmma to hiptensor
- replace this hard code with type_traits of f16